openflow15/flowstats.go

```diff
@@ -53,35 +53,36 @@ func (s *Stats) MarshalBinary() (data []byte, err error) {
 }
 
 func (s *Stats) UnmarshalBinary(data []byte) (err error) {
-	logrus.Debugf("Stats Data: %x", data)
+	klog.V(4).Infof("Stats Data: %x", data)
 	n := 2 // 2 bytes Reserved
 	s.Length = binary.BigEndian.Uint16(data[n:])
 	n += 2
-	logrus.Debugf("Stats Length: %d", s.Length)
+	klog.V(4).Infof("Stats Length: %d", s.Length)
 	for n < int(s.Length) {
 		var f util.Message
-		logrus.Debugf("Stats Field: %d", data[n+2]>>1)
+		klog.V(4).InfoS("Stats Field", "value", data[n+2]>>1)
 		switch data[n+2] >> 1 {
 		case XST_OFB_DURATION:
 			fallthrough
 		case XST_OFB_IDLE_TIME:
-			logrus.Debugf("Received TimeStatField n:%d", n)
+			klog.V(4).InfoS("Received TimeStatField", "n", n)
```

Comment:
Use a meaningful phrase to replace "n" ? Why only wants to log the bit index?

Reply:
> Use a meaningful phrase to replace "n" ? Why only wants to log the bit index?

This is legacy code. Maybe we clean up these loggings?

Reply:
Make sence.